packages/shared/src/evm.ts

```diff
@@ -1,21 +1,22 @@
+import { ChainflipNetwork } from '@chainflip/bitcoin';
+import { ChainflipChain } from '@chainflip/utils/chainflip';
 import { Signer } from 'ethers';
 import { getEvmChainId } from './consts';
-import { Chain, ChainflipNetwork, ChainflipNetworks } from './enums';
 import { assert } from './guards';
 
-export const assertIsEvmChain = (chain: Chain) => {
-  const evmChainId = getEvmChainId(chain, ChainflipNetworks.backspin);
+export const assertIsEvmChain = (chain: ChainflipChain) => {
+  const evmChainId = getEvmChainId(chain, 'backspin');
   assert(evmChainId, `Chain ${chain} is not an evm chain`);
 };
 
-export const assertIsCCMDestination = (chain: Chain) => {
+export const assertIsCCMDestination = (chain: ChainflipChain) => {
   if (chain === 'Solana') return;
   assertIsEvmChain(chain);
 };
```

Comment:
I think these 2 assertions are not used anywhere?

Reply:
actually all of the assertions in this file

Reply:
yeah probably a ton of dead ccm logic, will double check

Reply:
Would be nice to 💣  this

Reply:
yeeted